src/components/Patient/Utils.ts

```diff
@@ -17,3 +22,38 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: {
+  patientId: string;
+  consultationId?: string;
+}) => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: async ({
+      noteField,
+      thread,
+      replyTo,
+    }: {
+      noteField: string;
+      thread: 10 | 20;
```

Comment:
avoid duplicating type definitions. might break if things are changed in other places.

```suggestion
      thread: PatientNotesModel["thread"];
```